wgpu-hal/src/gles/device.rs

```diff
@@ -302,11 +350,7 @@ impl super::Device {
 
         // Create empty fragment shader if only vertex shader is present
         if has_stages == wgt::ShaderStages::VERTEX {
-            let version = match self.shared.shading_language_version {
-                naga::back::glsl::Version::Embedded { version, .. } => version,
-                naga::back::glsl::Version::Desktop(_) => unreachable!(),
-            };
-            let shader_src = format!("#version {} es \n void main(void) {{}}", version,);
+            let shader_src = format!("#version {} es \n void main(void) {{}}", glsl_version,);
```

Comment:
We do eventually want to support desktop glsl - panicing here I think is more useful than silently generating bad glsl

Reply:
I agree, this was old code that was just split apart and the panic still happens above.